contributing/content-style-guide.md

```diff
@@ -18,11 +18,51 @@ This guide includes standards we want to emphasize, likely because we've made de
 * [Images](#Images)
 * [Talk to us](#Talk-to-us)
 
+## Folder Structure and TOC
+
+The folder structure for the [docs.getdbt.com](https://github.com/dbt-labs/docs.getdbt.com) repository is organized into several high-level categories under the main `website` folder: blog, cypress, docs, functions, plugins, snippets, src, static.
+
+The table of contents (TOC) is managed in the `sidebar.js` file. You only need to edit the `sidebar.js` file when you are adding a new page or deleting an existing page. The `sidebar.js` file is the one that causes most of the merge conflicts because many technical writers are working on content daily. You will need to accept the changes from other contributors if you are committing a PR.
+
+Don't worry if you're not sure where in the TOC a new topic belongs. Do your best and when you submit your PR, the dbt Labs Documentation team will edit it and help to find the right placement.
+
+The right-hand TOC is created automatically when you add headings to a page.
+
+## Filenaming
+
+If you are adding a new file in the [docs.getdbt.com](https://github.com/dbt-labs/docs.getdbt.com) repository, it must be named following our naming conventions. MORE TO ADD
+
+## Using Markdown
```

Comment:
This whole section is awesome and very useful.  If/when we rearrange the sections I think this should be at the top.